modules/openy_demo_ahb/config/install/migrate_plus.migration.openy_demo_entity_ahb.yml

```diff
@@ -16,251 +16,110 @@ source:
-      field_ahb_video: 'https://www.youtube.com/watch?v=D4vzq664QLg'
-      field_ahb_pages: '/admin/structure/paragraphs_type'
-      field_ahb_visibility: 'include'
-    -
-      id: 19
-      field_ahb_title: 'Blogs - Anatomy'
-      field_ahb_description: |
-        <p>Can be found under Content (after filtering for blog content). It could be placed at the root of <a href="/content">/content</a> to be able to find easier, and/or add individual links to each section. Given there are a number of tutorials, suggest linking to the existing playlist on YouTube and have one link - <a href="https://www.youtube.com/watch?v=--gW0zj4wEc&list=PL_QVggMcFfKZZlVzNcVaIWCl4WwcOr9Jk">https://www.youtube.com/watch?v=--gW0zj4wEc&list=PL_QVggMcFfKZZlVzNcVaIWCl4WwcOr9Jk</a></p>
-      field_ahb_video: 'https://www.youtube.com/watch?v=PTZkgOb8CFE'
-      field_ahb_pages: '/node/add/blog'
-      field_ahb_visibility: 'include'
-    -
-      id: 20
-      field_ahb_title: 'Blogs - Overview'
+      id: 9
+      field_ahb_title: 'YMCA Website Services: Blogs'
       field_ahb_description: |
-        <p>Can be found under Content (after filtering for blog content). It could be placed at the root of <a href="/content">/content</a> to be able to find easier, and/or add individual links to each section. Given there are a number of tutorials, suggest linking to the existing playlist on YouTube and have one link - <a href="https://www.youtube.com/watch?v=--gW0zj4wEc&list=PL_QVggMcFfKZZlVzNcVaIWCl4WwcOr9Jk">https://www.youtube.com/watch?v=--gW0zj4wEc&list=PL_QVggMcFfKZZlVzNcVaIWCl4WwcOr9Jk</a></p>
-      field_ahb_video: 'https://www.youtube.com/watch?v=--gW0zj4wEc&t=7s'
+        <p>Blog posts display timely content, articles and news pieces tagged with one or more physical locations. <a href="https://ds-docs.y.org/docs/user-documentation/content-types/blog-post/" target="_blank">Learn more in our docs</a>.</p>
+      field_ahb_video: 'https://www.youtube.com/watch?v=--gW0zj4wEc&list=PL_QVggMcFfKZZlVzNcVaIWCl4WwcOr9Jk&index=1&t=10s'
       field_ahb_pages: '/node/add/blog'
       field_ahb_visibility: 'include'
     -
-      id: 21
+      id: 10
       field_ahb_title: 'YMCA Website Services: Taxonomy'
       field_ahb_description: |
-        <p>Can be found under Content (after filtering for blog content). It could be placed at the root of <a href="/content">/content</a> to be able to find easier, and/or add individual links to each section. Given there are a number of tutorials, suggest linking to the existing playlist on YouTube and have one link - <a href="https://www.youtube.com/watch?v=--gW0zj4wEc&list=PL_QVggMcFfKZZlVzNcVaIWCl4WwcOr9Jk">https://www.youtube.com/watch?v=--gW0zj4wEc&list=PL_QVggMcFfKZZlVzNcVaIWCl4WwcOr9Jk</a></p>
+        <p>Group pieces of related content together for tagging, filtering, sorting and grouping. <a href="https://ds-docs.y.org/docs/user-documentation/taxonomy/" target="_blank">Learn more in our docs</a>.</p>
```

Comment:
```suggestion
        <p>Group pieces of related content together for tagging, filtering, sorting, and grouping. <a href="https://ds-docs.y.org/docs/user-documentation/taxonomy/" target="_blank">Learn more in our docs</a>.</p>
```

Reply:
I know, Standford comma dilemma....

Reply:
https://github.com/open-y-subprojects/openy_demo_content/assets/238201/a8c49601-2bc9-415c-be02-a0ec019971ca

I copy and pasted from https://ds-docs.y.org/docs/user-documentation/taxonomy/... now I have another fix to fix.